signals.js

```diff
@@ -0,0 +1,35 @@
+const signals = {
+  SIGHUP: 1,
```

Comment:
These are in `os.constants.signals` (but have moved around since earlier versions of Node).